<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrganisationComboBoxPopup</class>
 <widget class="QWidget" name="OrganisationComboBoxPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>&amp;МО</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <widget class="COrganisationComboBoxListView" name="tblOrganisation"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>&amp;Поиск</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="QCheckBox" name="chkArea">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>&amp;Регион</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="CKLADRComboBox" name="cmbArea">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblInfis">
         <property name="text">
          <string>Код ИН&amp;ФИС</string>
         </property>
         <property name="buddy">
          <cstring>edtInfis</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="edtInfis"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="lblName">
         <property name="text">
          <string>&amp;Название содержит</string>
         </property>
         <property name="buddy">
          <cstring>edtName</cstring>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="edtName"/>
       </item>
       <item row="5" column="0" rowspan="2">
        <widget class="QLabel" name="lblPolisSerial">
         <property name="text">
          <string>Серия</string>
         </property>
         <property name="buddy">
          <cstring>edtPolisSerial</cstring>
         </property>
        </widget>
       </item>
       <item row="11" column="0" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QDialogButtonBox" name="buttonBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="standardButtons">
            <set>QDialogButtonBox::Apply|QDialogButtonBox::Reset</set>
           </property>
           <property name="centerButtons">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="edtINN"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblINN">
         <property name="text">
          <string>&amp;ИНН</string>
         </property>
         <property name="buddy">
          <cstring>edtINN</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="COrganisationComboBoxPopupPolicySerialEdit" name="edtPolisSerial"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lblOGRN">
         <property name="text">
          <string>ОГРН</string>
         </property>
         <property name="buddy">
          <cstring>edtOGRN</cstring>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="lblOKATO">
         <property name="text">
          <string>ОКАТО</string>
         </property>
         <property name="buddy">
          <cstring>edtOKATO</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QLineEdit" name="edtOGRN"/>
       </item>
       <item row="9" column="2">
        <widget class="QLineEdit" name="edtOKATO"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CKLADRComboBox</class>
   <extends>QComboBox</extends>
   <header>KLADR/kladrComboxes.h</header>
  </customwidget>
  <customwidget>
   <class>COrganisationComboBoxPopupPolicySerialEdit</class>
   <extends>QLineEdit</extends>
   <header>OrganisationComboBoxPopup.h</header>
  </customwidget>
  <customwidget>
   <class>COrganisationComboBoxListView</class>
   <extends>QListView</extends>
   <header>OrganisationComboBoxPopup.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>tblOrganisation</tabstop>
  <tabstop>chkArea</tabstop>
  <tabstop>cmbArea</tabstop>
  <tabstop>edtInfis</tabstop>
  <tabstop>edtName</tabstop>
  <tabstop>edtINN</tabstop>
  <tabstop>edtPolisSerial</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>chkArea</sender>
   <signal>clicked(bool)</signal>
   <receiver>cmbArea</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
